spec/calendar.html

```diff
@@ -622,24 +622,14 @@ <h1>DefaultMergeFields ( _fields_, _additionalFields_ )</h1>
       </p>
       <emu-alg>
         1. Let _merged_ be OrdinaryObjectCreate(%Object.prototype%).
-        1. Let _originalKeys_ be ? EnumerableOwnPropertyNames(_fields_, ~key~).
-        1. For each element _nextKey_ of _originalKeys_, do
-          1. If _nextKey_ is not *"month"* or *"monthCode"*, then
-            1. Let _propValue_ be ? Get(_fields_, _nextKey_).
-            1. If _propValue_ is not *undefined*, then
-              1. Perform ! CreateDataPropertyOrThrow(_merged_, _nextKey_, _propValue_).
-        1. Let _newKeys_ be ? EnumerableOwnPropertyNames(_additionalFields_, ~key~).
-        1. For each element _nextKey_ of _newKeys_, do
-          1. Let _propValue_ be ? Get(_additionalFields_, _nextKey_).
-          1. If _propValue_ is not *undefined*, then
-            1. Perform ! CreateDataPropertyOrThrow(_merged_, _nextKey_, _propValue_).
-        1. If _newKeys_ does not contain either *"month"* or *"monthCode"*, then
-          1. Let _month_ be ? Get(_fields_, *"month"*).
-          1. If _month_ is not *undefined*, then
-            1. Perform ! CreateDataPropertyOrThrow(_merged_, *"month"*, _month_).
-          1. Let _monthCode_ be ? Get(_fields_, *"monthCode"*).
-          1. If _monthCode_ is not *undefined*, then
-            1. Perform ! CreateDataPropertyOrThrow(_merged_, *"monthCode"*, _monthCode_).
+        1. Perform ? CopyDataProperties(_merged_, _fields_, « », « *undefined* »).
+        1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(%Object.prototype%).
+        1. Perform ? CopyDataProperties(_additionalFieldsCopy_, _additionalFields_, « », « *undefined* »).
+        1. NOTE: Every property of _additionalFieldsCopy_ is a data property with non-*undefined* value, but some property keys may be Symbols.
```

Comment:
what does the "but" add here?

Reply:
The mention of Symbol-keyed properties is intended to head off potential accidentally-normative refactors of the following line from [[OwnPropertyKeys]] to e.g. EnumerableOwnPropertyNames (which limits processing to String-keyed properties).